ros_gz_sim_demos/launch/imu.launch.xml

```diff
@@ -0,0 +1,14 @@
+<launch>
+  <gz_server
+    world_sdf_file="sensors.sdf"
+    use_composition="True"
+    create_own_container="True" />
+  <ros_gz_bridge
+    bridge_name="ros_gz_bridge"
+    config_file="$(find-pkg-share ros_gz_sim_demos)/config/imu.yaml"
+    use_composition="True" />
+  <node pkg="rqt_topic" exec="rqt_topic" args="-t" />
+  <!-- FIXME: Add once there's an IMU display for RViz2 -->
```

Comment:
The [rviz_imu_plugin](https://index.ros.org/p/rviz_imu_plugin/#rolling) package does provide this functionality.